spec/controllers/downloads_controller_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe DownloadsController do
+  describe '#show' do
+    let(:user) { FactoryGirl.create(:admin) }
+    let(:file_set) do
+      FactoryGirl.create(:file_set, user: user, content: File.open(fixture_path + '/files/video.mp4'))
+    end
+
+    context "when the user has access" do
+      before { sign_in user }
+
+      it 'sends the original file' do
+        get :show, :id => file_set.id
+        expect(response.body).to eq file_set.original_file.content
+      end
+
+      it 'sends the derivative' do
```

Comment:
I think this could be an issue since Sufia/CC don't have the service files and allow public users to download the original files. Could we set up another ticket for this matter as global support? We may need to determine what kind of intermediate files we need to have in DAMS5 first.